sympy/physics/units/systems/si.py

```diff
@@ -24,7 +24,7 @@
     katal, gray, becquerel, inch, liter, julian_year, gravitational_constant,
     speed_of_light, elementary_charge, planck, hbar, electronvolt,
     avogadro_number, avogadro_constant, boltzmann_constant,
-    stefan_boltzmann_constant, atomic_mass_constant, molar_gas_constant,
+    stefan_boltzmann_constant, atomic_mass_constant, electron_rest_mass_constant, molar_gas_constant,
```

Comment:
It looks like you imported this name but never defined it anywhere.

Reply:
imported from definitions via init?